TestCases/radiation/p1model/configp1.cfg

```diff
@@ -0,0 +1,130 @@
+% Author: Ruben Sanchez                                                        %
+% Institution: Chair for Scientific Computing, TU Kaiserslautern               %
+% Date: 2019-01-29                                                             %
+% File Version 7.0.1 "Blackbird"                                               %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+
+SOLVER= INC_NAVIER_STOKES
+KIND_TURB_MODEL= NONE
+MATH_PROBLEM= DIRECT
+RESTART_SOL= NO
+
+RADIATION_MODEL = P1
+MARKER_EMISSIVITY= ( upper, 0.0, lower, 0.0, left, 1.0, right, 1.0 )
+P1_INITIALIZATION = TEMPERATURE_INIT
+CFL_NUMBER_RAD = 10000
+TIME_DISCRE_RADIATION= EULER_IMPLICIT
+
+SCREEN_OUTPUT=(INNER_ITER, RMS_PRESSURE, RMS_VELOCITY-X, RMS_TEMPERATURE, RMS_RAD_ENERGY)
+
+INC_NONDIM = DIMENSIONAL
+
+ABSORPTION_COEFF = 0.2
+SCATTERING_COEFF = 0.0
+
+% ---------------- INCOMPRESSIBLE FLOW CONDITION DEFINITION -------------------%
+
+INC_DENSITY_MODEL= VARIABLE
+INC_ENERGY_EQUATION = YES
+INC_DENSITY_INIT= 0.00597782417156  % Ra ~ 1e6
+INC_TEMPERATURE_INIT= 288.15
+
+FLUID_MODEL= INC_IDEAL_GAS
+SPECIFIC_HEAT_CP= 1004.703
+MOLECULAR_WEIGHT= 28.96
+
+% --------------------------- VISCOSITY MODEL ---------------------------------%
+
+VISCOSITY_MODEL= CONSTANT_VISCOSITY
+MU_CONSTANT= 1.716e-5
+
+% --------------------------- THERMAL CONDUCTIVITY MODEL ----------------------%
+
+CONDUCTIVITY_MODEL= CONSTANT_CONDUCTIVITY
+KT_CONSTANT= 0.0246295028571
+
+% ----------------------- BODY FORCE DEFINITION -------------------------------%
+
+BODY_FORCE= YES
+BODY_FORCE_VECTOR= ( 0.0, -9.81, 0.0 )
+
+% ---------------------- REFERENCE VALUE DEFINITION ---------------------------%
+
+REF_ORIGIN_MOMENT_X = 0.25
+REF_ORIGIN_MOMENT_Y = 0.00
+REF_ORIGIN_MOMENT_Z = 0.00
+
+REF_AREA= 1.0
+
+% -------------------- BOUNDARY CONDITION DEFINITION --------------------------%
+
+MARKER_HEATFLUX= ( upper, 0.0, lower, 0.0 )
+MARKER_ISOTHERMAL= ( left, 461.04, right, 115.26 )
+MARKER_PLOTTING= ( upper, left, right, lower )
+MARKER_MONITORING= ( NONE )
+
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+
+NUM_METHOD_GRAD= WEIGHTED_LEAST_SQUARES
+CFL_NUMBER= 100
+CFL_ADAPT= NO
+
+CFL_ADAPT_PARAM= ( 1.5, 0.5, 15.0, 1e10)
+MAX_DELTA_TIME= 1E6
+
+RK_ALPHA_COEFF= ( 0.66667, 0.66667, 1.000000 )
+ITER= 101
+
+% ------------------------ LINEAR SOLVER DEFINITION ---------------------------%
+
+LINEAR_SOLVER= FGMRES
+LINEAR_SOLVER_PREC= ILU
+LINEAR_SOLVER_ILU_FILL_IN= 0
+LINEAR_SOLVER_ERROR= 1E-8
+LINEAR_SOLVER_ITER= 10
+
+% -------------------------- MULTIGRID PARAMETERS -----------------------------%
+
+MGLEVEL= 0
+MGCYCLE= V_CYCLE
+MG_PRE_SMOOTH= ( 1, 1, 1, 1 )
+MG_POST_SMOOTH= ( 0, 0, 0, 0 )
+MG_CORRECTION_SMOOTH= ( 0, 0, 0, 0 )
+MG_DAMP_RESTRICTION= 0.8
+MG_DAMP_PROLONGATION= 0.8
```

Comment:
I would trim the options to the bare minimum to highlight the specifics of the testcase.